src/pages/shop/Shop.css

```diff
@@ -0,0 +1,706 @@
+}
+
+.day-star1-image {
+    width: 13px;
+    height: 13px;
+}
+
+.day-star2-image {
+    width: 13px;
+    height: 13px;
+}
+
+.day-star3-image {
+    width: 13px;
+    height: 13px;
+}
+
+.day-star4-image {
+    width: 13px;
+    height: 13px;
+}
+
+.day-star5-image {
+    width: 13px;
+    height: 13px;
+}
+
+.day-reviews {
+    color: #000;
+    font-family: "Hiragino Mincho Pro";
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 300;
+    line-height: normal;
+}
+
+.day-size-buttons-container {
+    display: flex;
+    align-items: center;
+    gap: 16.271px;
+}
+
+.day-button-30ml {
+    display: flex;
+    align-items: center;
+    gap: 4.438px;
+}
+
+.day-button-image-30ml {
+    width: 12.358px;
+    height: 12.573px;
+    fill: #D9D9D9;
+    stroke-width: 1.479px;
+    stroke: #000;
+}
+
+.day-30ml-text {
+    color: #000;
+    font-family: "Hiragino Mincho Pro";
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 300;
+    line-height: normal;
+}
+
+.day-button-60ml {
+    display: flex;
+    align-items: center;
+    gap: 4.438px;
+}
+
+.day-button-image-60ml {
+    width: 12.358px;
+    height: 12.573px;
+    fill: #D9D9D9;
+    stroke-width: 1.479px;
+    stroke: #000;
+}
+
+.day-60ml-text {
+    color: #000;
+    font-family: "Hiragino Mincho Pro";
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 300;
+    line-height: normal;
+}
+
+.day-ViewProduct-button{
+    display: flex;
+    height: 39px;
+    padding: 4.438px 57.689px;
+    justify-content: center;
+    align-items: center;
+    gap: 7.396px;
+    align-self: stretch;
+    color: #000;
+    font-family: "Hiragino Mincho Pro";
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 300;
+    line-height: normal;
+}
+
+.night-cream-container {
+    display: flex;
+    width: 338.446px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 20px;
+}
+
+.night-cream-image {
+    width: 410.541px;
+    height: 410px;
+}
+
+.night-header-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    gap: 10px;
+    align-self: stretch;
+    flex-wrap: wrap;
+}
+
+.night-header {
+    flex: 1 0 0;
+    color: #000;
+    font-family: Baskerville;
+    font-size: 32px;
+    font-style: italic;
+    font-weight: 400;
+    line-height: normal;
+}
+
+.night-price-container {
+    color: #000;
+    font-family: "Hiragino Mincho Pro";
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 300;
+    line-height: normal;
+}
+
+.night-description {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 16px;
+    align-self: stretch;
+}
+
+.night-description-text {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 16px;
+    align-self: stretch;
+}
+
+.night-reviews-container {
+    display: flex;
+    align-items: flex-start;
+    gap: 5px;
+}
+
+.night-stars-container {
```

Comment:
I would try to combine some of these classes into one, just to make your file a little more condensed!